google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataSettings.java

```diff
@@ -375,6 +384,35 @@ public Builder setPrimingTableIds(String... tableIds) {
       return stubSettings.getPrimedTableIds();
     }
 
+    /**
+     * Enable latency based throttling for {@link BigtableDataClient#newBulkMutationBatcher(String)}
+     * with a target rpc latency. The number of allowed in-flight requests will be adjusted to reach
+     * the target bulk mutations rpc latency.
+     */
+    @BetaApi("Latency based throttling is not currently stable and may change in the future")
+    public Builder enableBatchMutationLatencyBasedThrottling(long targetRpcLatencyMs) {
+      stubSettings.bulkMutateRowsSettings().setLatencyBasedThrottling(true, targetRpcLatencyMs);
+      return this;
+    }
+
+    /**
+     * Disable latency based throttling for {@link
+     * BigtableDataClient#newBulkMutationBatcher(String)}.
+     */
+    @BetaApi("Latency based throttling is not currently stable and may change in the future")
+    public Builder disableBatchMutationLatencyBasedThrottling() {
+      stubSettings.bulkMutateRowsSettings().setLatencyBasedThrottling(false, null);
+      return this;
+    }
+
+    /**
+     * Gets if latency based throttling is enabled for {@link
+     * BigtableDataClient#newBulkMutationBatcher(String)}
+     */
+    @BetaApi("Latency based throttling is not currently stable and may change in the future")
+    public boolean isLatencyBasedThrottlingForBatchMutationEnabled() {
+      return stubSettings.bulkMutateRowsSettings().isLatencyBasedThrottlingEnabled();
+    }
```

Comment:
Same as above, if you expose the getter for one attribute, you should expose the other as well.